src/Form/atoms/SelectField.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useCallback, useState, SelectHTMLAttributes } from 'react';
+import styled, { css } from 'styled-components';
+import Label, { StyledLabel } from './Label';
+import { ISelectSizes } from '../index';
+import Icon, { IconWrapper } from '../../Icons/Icon';
+
+
+const Container = styled.div<{ styleSize?: ISelectSizes, }>`
+
+${({ styleSize }) => (styleSize === 'small') && css`
+  ${StyledLabel} {
+      span {
+        margin-bottom: 6px;
+      }
+    }
+  `}
+`;
+
+const StyledSelect = styled.select<{ styleSize?: ISelectSizes, activePlaceholder: boolean, selectWidth?: string }>`
+  box-sizing: border-box;
+  outline: none;
+  border-radius: 3px;
+  height: 40px;
+  padding: 0 25px 0 12px;
+  appearance: none;
+  width: ${({ selectWidth }) => selectWidth ? selectWidth : '220px'};
+
+  &::-ms-expand {
+    display: none;
+  }
+
+  ${({ theme }) => css`
+    border: 1px solid ${theme.styles.form.input.default.normal.borderColor};
+    font-family: ${theme.fontFamily.data};
+    ${theme.typography.form.input.value.normal};
+  `};
+
+  ${({ theme: { typography }, styleSize }) => (styleSize === 'small') && css`
+    height: 30px;
+    padding: 0 25px 0 5px;
+    ${typography.form.input.value.compact};
+  `};
+
+  ${({ theme, activePlaceholder }) => activePlaceholder && css`
+    ${theme.typography.form.input.placeholder.normal};
+  `};
+
+  ${({ theme, styleSize, activePlaceholder }) => (styleSize === 'small') && activePlaceholder && css`
+    ${theme.typography.form.input.placeholder.compact};
+  `};
+
+  option {
+    font-style: normal;
+    ${({ theme }) => css`
+      ${theme.typography.form.input.value.normal};
+    `};
+
+    ${({ theme: { typography }, styleSize }) => (styleSize === 'small') && css`
+      ${typography.form.input.value.compact};
+    `};
+  }
+`;
+
+const SelectWrapper = styled.div`
+  position: relative;
+  display: inline-block;
+
+  ${IconWrapper} {
+    position: absolute;
+    top: calc(50% - 7px);
+    right: 10px;
+  }
+`;
+
+interface ILabel {
+  htmlFor: string
+  labelText: string
+}
+
+interface OwnProps {
+  labelProps?: ILabel
+  styleSize?: ISelectSizes
+  placeholder?: string
+  selectWidth?: string
```

Comment:
Can we just use `width` for this as it's the Select component?

Reply:
I was trying to make it flexible like the input, I will just leave it with 100% and let it be adjust to the container.

Reply:
I think when we provide a `<select>` wrapped with the label then it's good to leave the select at 100%. If we're importing it directly I think there is use cases where it makes sense to allow an option width as you've described. 

By default it should just scale automatically but I can see times were it may be used in some flex based columns were we'd want to set the sizing. Perhaps in that case a max-width option would be good too. 

We can iterate later so feel free to make a decision you feel is best and we can adjust in the future as we use it and see how it's being used.